docs/plugin_developer_guide.md

```diff
@@ -1111,6 +1111,8 @@ You can find other examples in [tests](https://github.com/maxbrunsfeld/counterfe
 
 IBM Cloud CLI tends to be used globally. Both IBM Cloud CLI and its plug-ins should support globalization. We have enabled internationalization (i18n) for CLI's base commands with the help of the third-party tool "[go-i18n](https://github.com/nicksnyder/go-i18n)". To keep user experience consistent, we recommend plug-in developers follow the CLI's way of i18n enablement.
 
+Please install the *go-i18n* CLI for version 1.10.0. Newer versions of the CLI are no longer compatible with translations files prior to go-i18n@2.0.0. You can install the CLI using the command: `go install github.com/nicksnyder/go-i18n/goi18n@1.10.1`
```

Comment:
I'm not clear on why we are having users install the old version of go-i18n. Is this always necessary?

Reply:
Yes. The new version of the go-i18n generates translation files using a different json structure. I checked both versions again and got this result.

**Generated JSON from en_US in cli-dv-plugin using go-i18n CLI**

*Using V2*
```
{
  "Could not create the service instance.": {
    "hash": "sha1-xxxx",
      "other": "Could not create the service instance"
  }
  ...
}
```

*Using V1*
```
{
  "id": "Could not create the service instance",
  "translation": "Could not create the service instance"
}
```